client/sites-dashboard/components/no-sites-message.tsx

```diff
@@ -116,13 +133,17 @@ export const NoSitesMessage = ( { status, statusSiteCount }: SitesContainerProps
 
 	return (
 		<NoSitesLayout
-			title={ __( 'Create your first site' ) }
-			line={ __(
-				"It's time to get your ideas online. We'll guide you through the process of creating a site that best suits your needs."
-			) }
+			title={ <Title> { __( 'Create your first site' ) } </Title> }
```

Comment:
Is it good that we add a `div` component into the heading? Would it be more correct to remove it and style it in the context of `NoSitesLayout` component?

Reply:
I think it make sense to override/modify the underlying classes to style the child components.

Reply:
After the latest changes, I'm doing that only for the image as there was no other way to style it.

I had concerns about `div` and `p`, which we are adding into `h2` and `h3` to encapsulate styling. @zaguiini should we care about producing making sense HTML here? Or are we giving it up in favor of using styled components and not styling those elements using nested CSS selectors?

Reply:
I see @vykes-mac in another comment gave reasonable arguments about styling those `div` and `p` instead of nested classes:

> changing underlying css might result with our component stop working if the class names for the empty content is changed. I guess it's a tradeoff between modularity and simplicity and we seem to be lean more toward modularity.

Reply:
Makes a lot of sense @wojtekn